tests/integration/conftest.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+import hubble
+import pytest
+from docarray import DocumentArray
+from tests.constants import FINETUNER_LABEL
+
+import finetuner
+from finetuner.client.client import Client
+
+
+@pytest.fixture()
+def get_image_data(data_path='tests/resources/image_data/'):
+    left_da = DocumentArray.from_files(os.path.join(data_path, 'left/*.jpg'))
+    right_da = DocumentArray.from_files(os.path.join(data_path, 'right/*.jpg'))
```

Comment:
no please remove these images from resources, it makes repo too big to clone. You can generate some random images using numpy `np.random.rand(3, 224, 224)`